### PR TITLE
Simulate ping for the push simulation

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/scenari/PushPlatformValidationScenario.scala [hunk 1]
```diff
@@ -72,16 +73,20 @@ class PushPlatformValidationScenario(minMessagesInMailbox: Int,
...
+                5.0 -> sendMessage(recipientFeeder),
+                30.0 -> openArbitrary.openArbitrary,
+                10.0 -> flagUpdate,
+                15.0 -> getNewState,
+                30.0 -> exec())
+                .asInstanceOf[ChainBuilder]
+                .pause(minWaitDelay, maxWaitDelay),
+
+              exec(echoPingWs)
+                .pause(pingInterval))
```
Comment (Arsnael) [inline src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/scenari/PushPlatformValidationScenario.scala]: From what I understood, if I put multiple chains in an exec, they execute in parallel? But I'm not exactly sure... Guess we will see :)

Comment (chibenwa) [reply src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/scenari/PushPlatformValidationScenario.scala]: No idea...

Comment (Arsnael) [reply src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/scenari/PushPlatformValidationScenario.scala]: Conclusion: this was a terrible idea! Somehow it makes it opening much more websockets than the number of users running... Really odd. Trying to find other ways...

**Comment by Arsnael** [comment]
It seemed to work on my local with 100 users running for 15 minutes. No websocket connection lost. Will test tomorrow on charge-01 on a bigger scale

**Comment by Arsnael** [comment]
Jmap Push simulation with 3000 users, 15 mins injection, 45 mins run:

screenshot--2021 08 17-14_58_38

Getting some timeout on the ping response a few times but the websocket connection stayed alive until the end for all users :)

**Review by chibenwa — APPROVED** [review]

**Comment by chibenwa** [comment]
(Though I might not be surprised that using JMAP methods for the ping might contribute to put some pressure on James ;-) )

**Comment by Arsnael** [comment]
> (Though I might not be surprised that using JMAP methods for the ping might contribute to put some pressure on James ;-) )

Yeah totally agree on that, but it seems gatling does not manage the ping/pong mechanism at all regarding websockets so... couldn't find any better unfortunately^^'

**Comment by chibenwa** [comment]
Nope nope that's fine, just that we should just be aware of it.

**Pull request merged**